www/src/components/docsearch.min.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Not sure what's up with this file being here, but maybe we should add `.min.js` files to .eslintignore and not edit them with `/* eslint-disable */`?

Reply:
That's a good idea actually since minified JS files shouldn't be processed by eslint.